protos/grpc/testing/xdsconfig/control.proto

```diff
@@ -0,0 +1,47 @@
+// Copyright 2024 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package grpc.testing.xdsconfig;
```

Comment:
Quick question: will these protos eventually be added to the `xdsconfig` folder of https://github.com/grpc/grpc-proto/tree/master/grpc/testing?

Reply:
They will be significantly reworked and added there. There are plans to use control plane for other test cases. This is just an initial commit.

Reply:
Sounds good, thanks for explaining. Is there a tracking ticket for moving these protos to grpc/grpc-proto?